client-next/src/components/MapView/MapView.tsx

```diff
@@ -15,6 +15,13 @@ const initialViewState = {
   zoom: 4,
 };
 
+class PopupData {
```

Comment:
I would probably just use a regular type here and not use the class syntactic sugar.

Reply:
I have no idea how to do that but I will read it up.

Reply:
https://github.com/opentripplanner/OpenTripPlanner/pull/5602/commits/a18d77d4e924ba62212ede6394a2c79d2f585f76